.travis.yml

```diff
@@ -1,7 +1,7 @@
 dist: trusty
 sudo: required
 # xcode8 has jdk8
-osx_image: xcode8
+osx_image: xcode10.1
```

Comment:
How come we upgraded this?

Reply:
It aligns with `rules_scala_annex`.
https://github.com/higherkindness/rules_scala/blob/584e319f61a7c15360831b367c9a092570df9659/.travis.yml#L24